adding_metadata_sql.md

```diff
@@ -0,0 +1,186 @@
+
+_new species must be added to species table before referencing_
+
+```sql
+insert into cultivars (name, specie_id) 
+values ('RIL-CS27_(TX2910/(Macia/R07007)-CS44)-CSF1-PRF2-CS27', 2588)
+```
+
+### Step 5: Add new citations
+
+```sql
+insert into citations (author, year, title) 
+values ('Newcomb, Maria', 2016, 'Maricopa Agricultural Center Field Activities')
+```
+
+### Step 6: Associate experiments with sites
+
+```sql
+insert into experiments_sites (experiment_id, site_id) values ((select id from experiments where name = 'MAC Season 6: Sorghum BAP'),
+(select id from sites where sitename = 'MAC Field Scanner Season 6 Range 1 Column 1 E'))
+```
+
+When adding a new season for the TERRA REF project, a statement like the following can be used for associating experiments and sites since MAC Field Center sites are consistently named following the format `MAC Field Scanner Season x Range a Column b`.
+
+```sql
+insert into experiments_sites (experiment_id, site_id) 
+   select e.experiment_id, s.site_id 
+        from (select id as experiment_id from experiments where name = 'MAC Season 6: Sorghum BAP')  as e 
+     cross join 
+         (select id as site_id from sites where sitename like 'MAC Field Scanner Season 6%') as s
+```
+
+## Step 7: Associate experiments with treatments
+
+```sql
+insert into experiments_treatments (experiment_id, treatment_id) 
+values ((select id from experiments where name = 'MAC Season 6: Sorghum BAP'),
+(select id from treatments where name = 'MAC Season 6: Sorghum'))
+```
+When adding a new season for the TERRA REF project, a statement like the following can be used to associate experiments with treatments since experiment and treatment names are usually named following the format `MAC Season x: subexperiment name`.
+
+```sql
+insert into experiments_treatments (experiment_id, treatment_id) 
+   select e.experiment_id, t.treatment_id 
+        from (select id as experiment_id from experiments where name like 'MAC Season 6:%')  as e 
+     cross join 
+         (select id as treatment_id from treatments where name like 'MAC Season 6:%') as s
+```
+
+### Step 8: Associate sites with cultivars
+
+```sql
+insert into sites_cultivars (site_id, cultivar_id) 
+values ((select id from sites where sitename = 'MAC Field Scanner Season 8 Range 1 Column 1 E'),
+(select id from cultivars where name = 'Tiburon' and specie_id = 2588))
+```
+
+### Step 9: Associate sites with citations
+
+```sql
+insert into citations_sites (citation_id, site_id)
+values ((select id from citations where author = 'Newcomb, Maria' and year = 2016 and title = 'MAC Field Activities'),
+(select id from sites where sitename = 'MAC Field Scanner Season 6 Range 1 Column 1 E'))
+```
+When adding a new season for the TERRA REF project, a statement like the following can be used to associate citations with sites since MAC Field Center sites are consistently named following the format `MAC Field Scanner Season x Range a Column b`
```

Comment:
```suggestion

When adding a new season for the TERRA REF project, a statement like the following can be used to associate all citations with author "Newcomb, Maria" with all Season 6 sites since MAC Field Center sites are consistently named following the format `MAC Field Scanner Season x Range a Column b`

```